documentation/modules/creating-migration-quick-plan-new-source.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: PROCEDURE
+[id="creating-migration-quick-plan-new-source_{context}"]
+= Creating and running a migration quick plan with a new source provider
+
+You can create a new source provider as part of creating a migration _quick plan_.
+
+.Prerequisite
+
+* There is at least one user-defined destination provider.
+
+.Procedure
+
+. In the {ocp} web console, click *Providers for virtualization* and then click *Create Provider*.
+. Create a source provider following the instructions in xref:adding-source-providers[Adding Source providers].
++
+The *Provider details* page opens in the *Details* tab. This might take a few minutes for a vSphere source provider.
```

Comment:
I see that the 'few minutes' appears here as well, @RichardHoch did you see it taking few minutes to get to the Provider details page?

Reply:
I  just deleted this file -- it's not needed.